### PR TITLE
Ajax Security Improvement

### PR DESCRIPTION
**Pull request by kalpeshBhagat**
https://app.asana.com/0/905199306909631/1205468134063080

Adding proper permission check to the AJAX callbacks

### CODE REVIEW

File: admin/class-settings.php [hunk 1]
```diff
@@ -153,7 +153,9 @@ public function toggle_addon() {
 		if ( ! isset( $_POST['nonce'] ) || ! wp_verify_nonce( $_POST['nonce'], 'mbrs_toggle_addon' ) ) {
 			die();
 		}
-
+		if ( ! current_user_can( 'manage_options' ) ) {
+			die( 'You are not allowed to make these changes' );
```
Comment (m4munib) [inline admin/class-settings.php:157]: @kalpeshBhagat 

I think we can use the WP function to send proper JSON response as message is involved.

```
wp_send_json_error( array(
  'msg' => esc_html__( 'You are not allowed to make these changes.', 'members' )
) );
```

**Review by cartpauj — APPROVED** [review]
Thanks @kalpeshBhagat this looks good to me now.

@m4munib can you do a final review next week?

**Comment by m4munib** [comment]
@kalpeshBhagat 
Changes look good to me. 
IMHO: Maybe we can avoid the console.log call.

**Comment by m4munib** [comment]
Hi @kalpeshBhagat 
Please remove console.log() call from the settings.js file. Thanks!

**Comment by kalpeshBhagat** [comment]
> Hi @kalpeshBhagat Please remove console.log() call from the settings.js file. Thanks!

Updated the code

**Review by m4munib — APPROVED** [review]

**Pull request merged**